.gitignore

```diff
@@ -4,3 +4,5 @@
 # Dependencies
 composer.lock
 vendor
+/composer.phar
+/phpunit.phar
```

Comment:
@NelinD Good job :clap:
This pull request looks good, can you remove this 2 lines, please?
Then I'll merged it.